components/diagnosis-suggestion/api/pom.xml

```diff
@@ -84,4 +84,28 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
```

Comment:
Don't add this here, there's a special section in the root pom.xml. With one global clirr configuration, it's easy to have an overview of what changed. Otherwise, it will be hard to search for all the clirr rules in every pom file.